README.md

```diff
@@ -196,6 +196,14 @@ json.author do
 end
 ```
 
+Some languages/frameworks don't handle `null` well, in which case you can use `ignore_nil!`
```

Comment:
Is this better? https://github.com/rails/jbuilder/commit/1268ef4e0763ea72a0ffb1d57a1161eecd2ef20b